index.html

```diff
@@ -239,11 +239,11 @@ <h2>contact me</h2>
       If you have an application you are interested in developing, 
       a feature that you need built or a project that needs coding. I’d love to help with it 
     </p>
-    <form action="">
-      <input type="text" placeholder="enter your name">
-      <input type="emal" placeholder="enter your email">
-      <textarea name="" id=""  placeholder="write your message here"></textarea>
-      <button type="submit" class="btn">get in touch</button>
+    <form action="https://formspree.io/f/mzbwnraj" method="post">
+      <input required type="text" name="user-name" maxlength="30" placeholder="enter your name">
+      <input required type="email" name="user-email" placeholder="enter your email">
+      <textarea required name="user-message" id=""  maxlength="500" placeholder="write your message here"></textarea>
+      <button type="submit" class="btn" >get in touch</button>
```

Comment:
- [ ] Please kindly add appropriate styling according to the given template. 😄 

## Figma Design
![eeeee](https://user-images.githubusercontent.com/103386183/205915111-7601de4d-f2a3-48ec-8e93-8e4648b3530e.png)